src/GitHub.VisualStudio/Views/GitHubPane/PullRequestDetailView.xaml

```diff
@@ -8,8 +8,10 @@
                                     xmlns:prop="clr-namespace:GitHub.VisualStudio.UI;assembly=GitHub.VisualStudio.UI"
                                     xmlns:markdig="clr-namespace:Markdig.Wpf;assembly=Markdig.Wpf"
                                     xmlns:vsui="clr-namespace:Microsoft.VisualStudio.Shell;assembly=Microsoft.VisualStudio.Shell.14.0"
+                                    xmlns:theming="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Imaging"
                                     Background="{DynamicResource GitHubVsToolWindowBackground}"
                                     Foreground="{DynamicResource GitHubVsWindowText}"
+                                    theming:ImageThemingUtilities.ImageBackgroundColor="{Binding RelativeSource={RelativeSource Self}, Path=Background.Color}"
```

Comment:
This is important to ensure that the icons have the right background in all themes. `Transparent` doesn't work either (it seems to behave almost the same as white).

If this was set to `Transparent`, it would look like this on a dark theme:
![image](https://user-images.githubusercontent.com/11444821/41501675-2790d572-71a9-11e8-878e-6317a1704b80.png)

Correct behavior:
![image](https://user-images.githubusercontent.com/11444821/41501677-2951934c-71a9-11e8-9b85-aba194f6a230.png)

Reply:
Just ran into this on the PR review authoring view. I've tried moving this property to `PullRequestFilesView` and it seems to work fine. What am I missing?